.env.dev

```diff
@@ -2,7 +2,7 @@ ENV=dev
 APP_NAME=Whitebox | Development
 APP_NAME_CRON=Whitebox | Development
 
-DATABASE_URL=postgresql://postgres:postgres@localhost:5432/postgres
+DATABASE_URL=postgresql://postgres:postgres@localhost:5433/postgres
```

Comment:
This should not be committed. Either try running Postgres on the specified port or always exclude this file if you need to change it

Reply:
Correct, I forgot it! All fine now, thanks.